Editor/Window/ConnectToFleetInput.cs

```diff
@@ -24,15 +24,21 @@ public class ConnectToFleetInput : StatefulInput
         private VisualElement _fleetId;
         private Label _fleetIdText;
         private VisualElement _fleetStatus;
+        private readonly VisualElement _container;
         private readonly GameLiftFleetManager _fleetManager;
         private readonly StateManager _stateManager;
         private Button _cancelButton;
+        private StatusBox _connectToAnywhereStatusBox;
+        
+        private const string URLText = "View Logs";
+        private const string ErrorText = "An error occurred when trying to create the Anywhere fleet: ";
```

Comment:
The strings being used should come from the Strings factory (which are enums/constants), and not be hardcoded strings.

Reply:
Fixed